Day1/Kotlin/FizzBuzz.kts

```diff
@@ -0,0 +1,21 @@
+/**
+ * @author: d-l-mcbride
+ * @date: 10/11/2020
+ **/
+
+fun fizzBuzz (fizzList: List<Int>) {
+
+    var output: String
+
+    fizzList.forEach {
+        output = when {
+            it % 15 == 0 -> "FizzBuzz"
+            it % 5 == 0 -> "Buzz"
+            it % 3 == 0 -> "Fizz"
+            else -> "${it}"
+        }
+        println(output)
+    }
+}
+
+fizzBuzz((1..15).toList())
```

Comment:
If I'm not wrong, a top level declaration is required.

```suggestion
fun main() {
       fizzBuzz((1..15).toList())
}
```

Reply:
For Kotlin script file(.kts) stand alone functions work without the associated main function. I can make the change though.

Reply:
In that case you can leave it as it is.